index.html

```diff
@@ -2181,6 +2181,159 @@ <h4>Presentations Including Holder Claims</h4>
           </pre>
         </section>
       </section>
+
+      <section>
+        <h3>Data Schemas</h3>
+
+        <p>
+Data schemas are useful when enforcing a specific structure on a given
+collection of data. There are at least two types of data schemas that this
```

Comment:
```suggestion
serialization of information. There are at least two types of data schemas that this
```

Reply:
Ugh, rebase put this text into this PR... but it has nothing to do with this PR -- this text is already in the main part of the spec via @decentralgabe's PR. 

Your input is valid, @OR13, but it's modifying text that has nothing to do w/ this PR due to the rebase. Can we deal w/ these changes in another PR?

I can rebase again to drop this text, but all of your change suggestions will be dropped if I do that, @OR13. 

Would you be averse to NOT taking these changes in this PR, but rather in a different PR focused on the Data Schemas section? This is text that @decentralgabe authored, so he'll have to write that PR. Can you raise such a PR @decentralgabe?

Reply:
rebase strikes again! 

The rest of my review may not be relevant after this is cleaned.